ref_builder/otu/update.py

```diff
@@ -201,8 +201,15 @@ def exclude_accessions_from_otu(
     accessions: list[str],
 ):
     """Take a list of accessions and add them to an OTU's excluded accessions list."""
+    otu_logger = logger.bind(taxid=otu.taxid, otu_id=str(otu.id), name=otu.name)
+
+    excluded_accessions = {}
     for accession in accessions:
-        repo.exclude_accession(otu.id, accession)
+        if accession in otu.excluded_accessions:
+            otu_logger.warning(f"{accession} is already excluded.")
+        excluded_accessions = repo.exclude_accession(otu.id, accession)
+
+    otu_logger.info(f"Excluded accessions: {excluded_accessions}")
```

Comment:
* A better separation of concerns would have `Repo` handling this check and raising and error if the accession is already excluded.
* Alternatively, it could be a no-op if one of the accessions is already excluded. I'm leaning this way, but don't really care.
* I found `exclude_accessions_from_otu()`. Is this not kind of doing the same thing without error messages.